mercat/wasm/src/lib.rs

```diff
@@ -0,0 +1,570 @@
+use codec::{Decode, Encode};
+use mercat::{
+    account::{convert_asset_ids, AccountCreator},
+    asset::AssetIssuer,
+    cryptography_core::{
+        asset_proofs::{CipherText, CommitmentWitness, ElgamalPublicKey, ElgamalSecretKey},
+        curve25519_dalek::scalar::Scalar,
+        AssetId,
+    },
+    transaction::{CtxMediator, CtxReceiver, CtxSender},
+    Account as MercatAccount, AccountCreatorInitializer, AssetTransactionIssuer, EncryptedAmount,
+    EncryptionKeys, FinalizedTransferTx, InitializedAssetTx, InitializedTransferTx,
+    MediatorAccount as MercatMediatorAccount, PubAccount as MercatPubAccount, PubAccountTx,
+    SecAccount, TransferTransactionMediator, TransferTransactionReceiver,
+    TransferTransactionSender,
+};
+use rand_core::OsRng;
+use serde::Serialize;
+use serde_json;
+use std::convert::Into;
+use wasm_bindgen::prelude::*;
+use wasm_bindgen::JsValue;
+
+// ------------------------------------------------------------------------------------
+// -                                  Type Definitions                                -
+// ------------------------------------------------------------------------------------
+
+/// A hex string without the "0x". Values of this type are plain values (i.e., not encrypted).
+pub type PlainHex = String;
```

Comment:
I think you can simply use `String` because plain hex strings are typed as `String` in the function signature
```rust
fn create_secret_account(rng: &mut OsRng, ticker_id: String)
```
In general, in cases when a type alias looks useful, it's very likely that a newtype `struct T(U)` could be an even better idea because type conversions would have to be explicit in the code.

But in this particular case I think a simple `String` is enough because this is essentially the unchecked JS input type.